docs/beginner/tutorial2-swapchain/README.md

```diff
@@ -4,6 +4,8 @@
 For convenience we're going to pack all the fields into a struct, and create some methods on that.
 
 ```rust
+use winit::Window;
+
 // main.rs
```

Comment:
This should go above `use winit::Window;` as it's used to tell what file we're currently changing.